Yona/Yona/DataService/Users.swift

```diff
@@ -64,6 +65,11 @@ struct Users{
                     foundIncludeLink = true
                 }
                 
+                if let yonaOpenAppEventLink = links[YonaConstants.jsonKeys.yonaOpenAppEventLink],
+                    let hrefyonaOpenAppEventLink = (yonaOpenAppEventLink as? [String : String])?[YonaConstants.jsonKeys.hrefKey] {
```

Comment:
Use a capital Y for Yona, as that begins a new word in the variable name: ``hrefYonaOpenAppEventLink``.

Reply:
Done.